uber/migration.py

```diff
@@ -0,0 +1,87 @@
+from collections import OrderedDict
+from os.path import abspath, dirname, join
+
+from alembic import command
+from alembic.config import Config as AlembicConfig
+from alembic.script import ScriptDirectory
+
+from sideboard.internal.imports import plugin_dirs
+
+
+# Path to "alembic.ini", cached here for convenience
+alembic_ini_path = abspath(join(plugin_dirs['uber'], 'alembic.ini'))
```

Comment:
FWIW, saying

```python
from sideboard.internal.imports import plugin_dirs
plugin_dirs['uber']
```

is the same as saying

```python
from uber.config import _config
_config['root']
```

Calling ``sideboard.lib.parse_config`` a ``root`` and ``module_root`` config options into the resulting ``ConfigObj`` dictionary.

I don't have a strong preference about which of those is better.  Either way we're relying on the internals of a codebase, whether it's the ``uber`` plugin or the ``sideboard.internal.imports`` module.